lib/ex_unit/lib/ex_unit.ex

```diff
@@ -312,6 +312,19 @@ defmodule ExUnit do
     ExUnit.Runner.run(config, nil)
   end
 
+  @doc """
+  Allows users to configure functions to run after the test suite has finished.
+  Functions passed in will be executed after all tests have finished running. If
+  `after_suite/1` is called multiple times, the functions will be executed in
+  reverse order (i.e. the last function given will be executed first).
+  """
+  @spec after_suite(fun) :: :ok
+  def after_suite(function) do
+    current_callbacks = Application.fetch_env!(:ex_unit, :after_suite)
+    configure(after_suite: [function | current_callbacks])
```

Comment:
I think `configure` returns `:ok` (it uses `Enum.each/2` which always returns `:ok`). 

Probably out of the scope of this PR, but I would include the spec `@spec configure(Keyword.t()) :: :ok` in the `configure/1` function :)

Reply:
Yeah, I saw a couple of functions here missing typespecs, so I was going to go and add them in a separate PR.